app/Models/CourseModel.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class CourseModel extends Model
+{
+    protected $table = 'courses';
+    protected $primaryKey = 'id';
```

Comment:
Not needed. It is automatically implied by Laravel, as already used in the other models.